Sources/StreamChat/Database/DatabaseSession.swift

```diff
@@ -400,11 +400,17 @@ extension DatabaseSession {
             return
         }
         
-        try saveMessage(
+        let savedMessage = try saveMessage(
             payload: messagePayload,
             channelDTO: channelDTO,
             syncOwnReactions: false
         )
+
+        if payload.eventType == .messageUpdated {
+            savedMessage.quotedBy.forEach { message in
+                message.updatedAt = savedMessage.updatedAt
+            }
+        }
```

Comment:
@evsaev @b-onc This is the only solution right now I've found to do this on DB Level. The issue here is that the update is triggered ~1s after, and this causes the table view to jump twice (because of the nature of reloading rows in table view)

Reply:
@nuno the trick with `id` should work, actually, we do a similar thing with channel reads 🤔

```
override func willSave() {
        super.willSave()
        
        // When the read is updated, we need to propagate this change up to holding channel
        if hasPersistentChangedValues, !channel.hasChanges {
            // this will not change object, but mark it as dirty, triggering updates
            channel.cid = channel.cid
        }
    }
```

Reply:
> update is triggered ~1s
Would you record a video? Are you saying the parent message update is propagated immediately while quote updates are delayed?

Reply:
https://user-images.githubusercontent.com/12814114/146810049-19b6dac6-6374-43db-b8f0-68e6b4afadba.mp4

@evsaev ☝️

Reply:
Ok, I see 😕 But customers are waiting for the fix, and handling it on DB level is definitely the right way. So even though it feels broken, we'd better jump than keep quote replies outdated. But it also means glitches on the message list become more visible/easy -> the sooner we give love to message list the better